lib/presentation/util/helper/responsive_widget.dart

```diff
@@ -40,6 +40,12 @@ class ResponsiveWidget extends StatelessWidget {
   static final int minMediumWidth = 600;
 
   static final double horizontalPaddingLargeWidth = 132.0;
+  static final double horizontalMarginMediumScreen = 144.0;
+  static final double verticalMarginMediumScreen = 234.0;
+  static final double horizontalMarginLargeScreen = 234.0;
+  static final double verticalMarginLargeScreen = 144.0;
```

Comment:
how to define that number?

Reply:
I have determined those values through our design file.